package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "devDependencies": {
     "babel-cli": "^6.24.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
```

Comment:
doesn't this need to be a hard dep?

Reply:
@evan-scott-zocdoc I double checked to be sure, but yeah this is only needed for test to work.

Method `path.isClassProperty()` works without this dependency too. In our use case we are doing the babel transpilation for styled-components before Typescript with `@babel/plugin-syntax-typescript` + some more (see https://github.com/gaearon/react-hot-loader#add-babel-before-typescript)